### PR TITLE
Kickoff api

### PR DESCRIPTION
**Pull request by ivoguzman23**
## Summary

Rubocop and CircleCI installation and configuration

This was an old card but I had problems, now is working fine.
Rubocop, Rubocop-rails gem installation, CircleCI project configuration.

## Screenshots

image

image

## TRELLO Card

https://trello.com/c/DeWRiBNT/13-configurar-circleci-o-si-no-travisci

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      # Run Tests!
+      # - run:
+      #     name: Run Tests
+      #     command: bundle exec rspec 
+
+workflows:
+  version: 2
+  build:
+    jobs:
+      - build
```
Comment (lucasenriquez) [inline .circleci/config.yml]: Add a newline at the end of the file

File: .env [hunk 2]
```diff
@@ -0,0 +1 @@
+DEVISE_JWT_SECRET_KEY=44a238d54045d8fe4a9a9cf3fd5a530ebb77d6122aff436d212917c65919ccda9e106ee1cbcd335232924657288c406433d80d3ed4aa3c8d675664d115421e00
```
Comment (lucasenriquez) [inline .env:1]: This file should not be here

File: .rubocop.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+Style/Documentation:
+  Enabled: false
```
Comment (lucasenriquez) [inline .rubocop.yml]: Add a newline at the end of the file

File: Gemfile [hunk 4]
```diff
@@ -26,37 +25,39 @@ gem 'jbuilder', '~> 2.5'
...
+gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
+
+gem 'devise'
+gem 'devise-jwt', '~> 0.6.0'
+gem 'dotenv-rails', groups: %i[development test]
+gem 'ffi', '~> 1.15.3'
+gem 'rubocop', '~> 1.17', require: false
+gem 'rubocop-rails', require: false
+gem 'execjs'
+gem 'therubyracer'
```
Comment (lucasenriquez) [inline Gemfile]: Add a newline at the end of the file

**Review by lucasenriquez — CHANGES_REQUESTED** [review]

File: app/controllers/application_controller.rb [hunk 5]
```diff
@@ -1,2 +1,26 @@
+# frozen_string_literal: true
+
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :null_session
+
+  def render_resource(resource)
+    if resource.errors.empty?
+      render json: resource
+    else
+      validation_error(resource)
```
Comment (melissaregnier) [inline app/controllers/application_controller.rb]: This can be made simpler with an early return and an inline `if`.

**Review by melissaregnier — CHANGES_REQUESTED** [review]

File: app/controllers/application_controller.rb [hunk 6]
```diff
@@ -1,2 +1,24 @@
+# frozen_string_literal: true
+
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :null_session
+
+  def render_resource(resource)    
+    #SE detiene el codigo en el return
```
Comment (melissaregnier) [inline app/controllers/application_controller.rb:7]: this comment is not needed

File: app/controllers/application_controller.rb [hunk 7]
```diff
@@ -1,2 +1,24 @@
...
+    return render json: resource if resource.errors.empty?
```
Comment (melissaregnier) [inline app/controllers/application_controller.rb:8]: add an empty line after an early return

**Review by melissaregnier — CHANGES_REQUESTED** [review]

**Pull request merged**